src/elem.js

```diff
@@ -182,6 +156,15 @@ function el(name, attrs, children) {
         }
       });
 
+      _.each(_.keys(attrs), function(key) {
+        var keyName = _.dasherize(key);
+        if (_.startsWith(keyName, 'on')) {
+          _.on(element, [keyName.toLowerCase().replace('on', '')], function() {
```

Comment:
With that one, we should be able to write stuff like <button onClick={...}> which is better JSX style